command_line/find_bad_pixels.py

```diff
@@ -30,6 +32,9 @@
 image_range = None
   .type = ints(value_min=0, size=2)
   .help = "Image range for analysis e.g. 1,1800"
+nproc = 1
+  .type = int
+  .help = "Number of processors to use for bad pixel finding"
```

Comment:
```suggestion
    .help = "The number of processes to use."
```
Probably strictly _processes_ not _processors_, and might as well use the help string as used elsewhere.